spec/models/course_spec.rb

```diff
@@ -78,7 +78,111 @@
     end
   end
 
-  describe '#get_assignment_list' # TODO
+  def create_assignment_csv_string(a)
+    # returns a csv formatted string for an assignment where each attribute
+    # specified by Assignment::DEFAULT_FIELDS appears in the same order as initialized
+    # and is comma separated.
+
+    # a is an assignment stored in the database
+
+    "#{a.short_identifier},#{a.description},#{a.due_date},#{a.message}," \
```

Comment:
For CSV string creation specifically, you don't want to manually create a string because you have to worry about escaping special characters (like commas) and double quotes. Instead you should create an array of attributes and then use a Ruby method to convert that array into a CSV string (see https://rubydoc.org/stdlib/csv/Array:to_csv).